### PR TITLE
Added ACI-CNI 5.2.7.1 templates

### PR DESCRIPTION
**Pull request by LavanyaVaddavalli**
https://github.com/rancher/rancher/issues/41762

### CODE REVIEW

**Comment by LavanyaVaddavalli** [comment]
Testing is under progress

File: pkg/rke/k8s_rke_system_images.go [hunk 1]
```diff
@@ -8418,6 +8418,50 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/mirrored-pause:3.6",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.21.1",
 		},
+		// Enabled out of band post v2.7.2
```
Comment (snasovich) [inline pkg/rke/k8s_rke_system_images.go]: Minor update to comment, same for the other introduced versions

```suggestion
		// Enabled with v2.7.5
```

File: pkg/rke/templates/templates.go [hunk 2]
```diff
@@ -223,13 +224,16 @@ func LoadK8sVersionedTemplates() map[string]map[string]string {
...
 			">=1.24.0-rancher0 <1.24.8-rancher1-1":     aciv523,
 			">=1.24.8-rancher1-1 <1.24.9-rancher1-1":   aciv5234,
 			">=1.24.9-rancher1-1 <1.24.13-rancher2-1":  aciv5235,
-			">=1.24.13-rancher2-1 <1.25.0-rancher0":    aciv5236,
+			">=1.24.13-rancher2-1 <1.24.13-rancher2-2": aciv5236,
+			">=1.24.13-rancher2-2 <1.25.0-rancher0":    aciv5271,
 			">=1.25.0-rancher0 <1.25.9-rancher2-1":     aciv5235,
-			">=1.25.9-rancher2-1":                      aciv5236,
+			">=1.25.9-rancher2-1 <1.25.9-rancher2-2":   aciv5236,
+			">=1.25.9-rancher2-2":                      aciv5271,
```
Comment (snasovich) [inline pkg/rke/templates/templates.go:236]: With this range change for `v1.26.4-rancher2-1`, `aciv5271` template will be used yet images version `5.2.3.6` are referenced in the definition of `v1.26.4-rancher2-1`.
I understand that ACI-CNI is not tested for 1.26, but I think the best option here is to bump versions to 5.2.7.1 for `v1.26.4-rancher2-1` to be consistent. Another rational is it won't make it worse and there is a higher chance of newer version of ACI-CNI plugin working on 1.26.

**Review by snasovich — CHANGES_REQUESTED** [review]
Couple comments to address, otherwise LGTM

**Comment by LavanyaVaddavalli** [comment]
@snasovich Addressed comments in e06af2e5da58f9659b0698a2f320d2acddea1655. Added aci cni version bump to 1.26 k8s version as well

**Review by snasovich — DISMISSED** [review]
Approved, but also hinging on RKE1 PR https://github.com/rancher/rke/pull/3258

**Review by snasovich — APPROVED** [review]

**Review by kinarashah — APPROVED** [review]

**Pull request merged**